madgiraffestudios/src/index.js

```diff
@@ -1,9 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import './index.css';
 import App from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import * as serviceWorker from './serviceWorker';
+import NavigationBar from './components/Navigation/NavigationBar'
+//import About from './components/About/About';
+import { WORK } from './Constants/routes';
 
+const routing = (
```

Comment:
- [ ] please remove extra line spaces and commented code

Reply:
spacing fixed